src/coreComponents/dataRepository/xmlWrapper.hpp

```diff
@@ -204,7 +208,7 @@ readAttributeAsType( T &, string const &, xmlNode const &, U const & )
  * @param[in] name       the name of the xml attribute to process
  * @param[in] targetNode the xml node that should contain the attribute
  * @param[in] defVal     default value of @p rval (or entries of @p rval, if it is an array)
- * @return boolean value indicating whether the value was successfully read from XML.
+ * @return true
```

Comment:
Why this change to the doc? The previous one seemed correct to me.

Reply:
It is always true for this SFINAE implementation.